doc/backend_vars.asciidoc

```diff
@@ -46,6 +46,7 @@ UPLOAD_INACTIVITY_TIMEOUT;integer;300;Specifies the inactivity timeout in second
 NO_DEPRECATE_BACKEND_$backend;boolean;0;Only warn about deprecated backends instead of aborting
 XRES;integer;1024;Resolution of display on x axis. Sets the resolution of the video encoder, and in qemu, the initial console resolution when OFW is set (Power and SPARC), and the EDID resolution for devices that support EDID
 YRES;integer;768;Resolution of display on y axis. Sets the resolution of the video encoder, and in qemu, the initial console resolution when OFW is set (Power and SPARC), and the EDID resolution for devices that support EDID
+VIDEO_ENCODER_BLOCKING_PIPE;boolean;0;Whether the pipe for writing data to the video encoder should be blocking or not
```

Comment:
It would be good to state what effect this has (advantages/disadvantages). And what exactly do you mean with "better user experience"?

Reply:
By better user experience I'm referring to the fact that in the case I had, the live view would not update anywhere close to real time as isotovideo was attempting to send ~4M of data in 64K chunks. Making this blocking meant that one could follow the progress of a test run in the live view as the next test wouldn't proceed until the full image had at least been sent over to the videoencoder.

As for advantages and disadvantages, it's hard to say exactly, one big advantage for me would be the working live view. Without sinking even more time into understanding the code it would be difficult for me personally to work out all of the side effects this may have.

Reply:
```suggestion
VIDEO_ENCODER_BLOCKING_PIPE;boolean;0;Whether the pipe for writing data to the video encoder should be blocking or not; making it blocking might allow following the live view in realtime despite large screenshot file sizes but it is not a well tested configuration
```

Reply:
Looks good, I've added it